### PR TITLE
Storybook install part 1

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Some suggestions and structure changes

# Needed By (Date)
- Whenevs
- 
# Urgency
- None
 
# Steps to Test

1. Read code
2. Review preview site build

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,4 +1,11 @@
 
 export const parameters = {
   actions: { argTypesRegex: "^on[A-Z].*" },
-}
\ No newline at end of file
+  html: {
+    prettier: {
```
Comment (yvonnetangsu) [inline .storybook/preview.js:5]: Nice stuff! I thought we're getting rid of prettier though? Or you want to keep it for html?

**Review by yvonnetangsu — APPROVED** [review]
Preview is awesome - GTG! I'm sure we'll keep tweaking stuff.

File: .storybook/preview.js (hunk 1 above)
Comment (sherakama) [reply .storybook/preview.js:5]: That is just the plugin config for the HTML output
https://storybook.js.org/addons/@whitespace/storybook-addon-html

Needs more tweaking but we can do that later.

Comment (yvonnetangsu) [reply .storybook/preview.js:5]: Cool - that HTML tab is really nice

**Pull request merged**